src/controller.js

```diff
@@ -615,6 +615,8 @@ export default ['$scope', '$element', function ($scope, $element) {
           propsString = propsString.replaceAll('$(vDim)', `'${dimName}'`);
           propsString = propsString.replaceAll('$(vDimValue)', `'${dimValue}'`);
           props = JSON.parse(propsString);
+          props.showTitles = true;
```

Comment:
since this is the same as the case above you might as well move it below the if else